### PR TITLE
fix: datatype of an `id` column

### PR DESCRIPTION
**Pull request by nathane**
fix the bug I created in #48

### CODE REVIEW

File: src/Generators/MigrationGenerator.php [hunk 1]
```diff
@@ -55,12 +55,10 @@ protected function buildDefinition(Model $model)
         /** @var \Blueprint\Models\Column $column */
         foreach ($model->columns() as $column) {
             $dataType = $column->dataType();
-            if ($column->name() === 'id' && $dataType !== 'uuid') {
+            if ($column->name() === 'id' && $dataType === 'id') {
                 $dataType = 'bigIncrements';
-            } elseif ($column->dataType() === 'id') {
+            }elseif ($column->name() !== 'id' && $dataType === 'id') {
```
Comment (jasonmccreary) [inline src/Generators/MigrationGenerator.php:60]: This is unnecessary given the `elseif`

Comment (nathane) [reply src/Generators/MigrationGenerator.php:60]: it's needed to build relationships. `$column->name() = 'user_id'` and `$dataType === 'id'`

**Comment by jasonmccreary** [comment]
Thanks all!

**Comment by msonowal** [comment]
@jasonmccreary 
Can you tag a release

**Pull request merged**